pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +124,18 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if errors.IsNotFound(err) {
+					runningStrategy, invalid := vm.RunStrategy()
+					if invalid != nil {
+						return invalid
```

Comment:
Very unlikely that we ever get here, but I think we should return a better error here. I'd suggest just the generic `return fmt.Errorf("Error pausing VirutalMachineInstance %s: %v", vmiName, err)`. Wdyt?

Reply:
Sure very unlikely, maybe we can also remove it in RunStrategy....